README.md

```diff
@@ -2,6 +2,10 @@
 
 GOG Galaxy 2.0 Community integration for Steam.
 
+## Installation
+- Replace the content of the folder created by GoG <br> e.g. `C:\Users\<User>\AppData\Local\GOG.com\Galaxy\plugins\installed\steam_ca27391f-2675-49b1-92c0-896d43afa4f8`
```

Comment:
it is not necessary to keep the same name of the folder, the key is to have ONE plugin for given platform (steam) in folder `installed` AND have its `manifest.json` file in the root of that folder (no nested directories).

But what you wrote works and makes perfect sense. It is also eaiser to understand, so lets keep this version, but add a note that "you can rename the `steam_ca...` as a last step (may be useful for debugging).

Reply:
Thanks! I was not aware of that. My text is only easier to understand if the plugin has already been installed by GoG previously. What will happen, if this is no the case and the user creates a new folder including the `manifest.json`, and then installs the plugin via GoG? Will GoG create a new folder and hence break the manual installation?

Reply:
> My text is only easier to understand if the plugin has already been installed by GoG previously.
True. Would you like to rephrase?

>  What will happen, if (...)

In that case you see plugin as installed in GOG Galaxy and clicking Connect won't download latest version from Github.

The exception is when the version provided in `manifest.json` is smaller than the latest version on Github. In such case Galaxy will download newer version in the background to `downloaded` directory and replace the plugin build after next Galaxy restart. The directory name will be replaced with standard `steam_ca27391f-2675-49b1-92c0-896d43afa4f8` (<platform_<guid_from_manifest>).